### PR TITLE
[ Docs] Update bare metal setup instructions

### PR DESCRIPTION
**Pull request by natrius**
Added some info to the baremetal installation and changed it up a bit for an hopefully easier installation experience

## Proposed change

Just flushed out the baremetal installation instructions so future installations hopefully will result in less questions like mine.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Other (please explain)

Clarifications

## Checklist:

- [x] I have read & agree with the [contributing guidelines](https://github.com/paperless-ngx/paperless-ngx/blob/main/CONTRIBUTING.md).
- [ ] If applicable, I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers.
- [ ] If applicable, I have checked that all tests pass, see [documentation](https://docs.paperless-ngx.com/development/#back-end-development).
- [ ] I have run all `pre-commit` hooks, see [documentation](https://docs.paperless-ngx.com/development/#code-formatting-with-pre-commit-hooks).
- [x] I have made corresponding changes to the documentation as needed.
- [x] I have checked my modifications for any breaking changes.

### CODE REVIEW

File: docs/setup.md [hunk 1]
```diff
@@ -401,7 +423,9 @@ supported.
     sudo -Hu paperless python3 manage.py runserver
     ```
 
-    and pointing your browser to <http://localhost:8000/>.
+    and pointing your browser to <http://localhost:8000/> if 
+    using the browser from the same devices as paperless is installed.
```
Comment (shamoon) [inline docs/setup.md:427]:
```suggestion
    and pointing your browser to http://localhost:8000 if 
    accessing from the same devices on which paperless is installed.
```

File: docs/setup.md [hunk 2]
```diff
@@ -452,6 +476,14 @@ supported.
         For instructions on how to use nginx for that,
         [see the instructions below](/setup#nginx).
 
+    !!! warning
+
+        If celery won't start (check with 
+        `sudo systemctl status paperless-task-queue.service` for 
+        paperless-task-queue.service und paperless-scheduler.service
```
Comment (shamoon) [inline docs/setup.md:483]:
```suggestion
        paperless-task-queue.service and paperless-scheduler.service
```

File: docs/setup.md [hunk 3]
```diff
@@ -314,14 +314,30 @@ supported.
...
+5.  Get the release archive from
+    <https://github.com/paperless-ngx/paperless-ngx/releases> for example with 
+    ```shell-session
+    curl -O -L https://github.com/paperless-ngx/paperless-ngx/releases/download/v1.10.2/paperless-ngx-v1.10.2.tar.xz
+    ```
+    Extract the archive with 
+    ```shell-session
+    tar -xf paperless-ngx-v1.10.2.tar.xz
+    ```
+    and copy the content to the
```
Comment (shamoon) [inline docs/setup.md:333]:
```suggestion
    and copy the contents to the 
```

File: docs/setup.md [hunk 4]
```diff
@@ -314,14 +314,30 @@ supported.
...
+    curl -O -L https://github.com/paperless-ngx/paperless-ngx/releases/download/v1.10.2/paperless-ngx-v1.10.2.tar.xz
+    ```
+    Extract the archive with 
+    ```shell-session
+    tar -xf paperless-ngx-v1.10.2.tar.xz
+    ```
+    and copy the content to the 
+    home folder of the user you created before (`/opt/paperless`).
+    
+    Optional: If you clone the git repo as it is, you also have to
```
Comment (shamoon) [inline docs/setup.md:336]:
```suggestion
    Optional: If you cloned the git repo, you will have to
```

**Review by shamoon — CHANGES_REQUESTED** [review]
I mostly made grammar changes etc. I will let others comment on the content but looks OK to me

**Review by principis — DISMISSED** [review]
Looks good! I have a suggestion to help updating:

- Extract to a subdirectory, for example: `/opt/paperless/paperless-ngx-v1.11.2`
- Create a `paperless-latest` symlink to the extracted directory: `paperless-latest -> paperless-ngx-v1.11.2`
- Change all services: `WorkingDirectory=/opt/paperless/paperless-latest/src`

This allows for easy upgrading by just changing the symlink.

**Comment by natrius** [comment]
Hm, that could be mentioned in an "Additional tips" or something like that but probably should not be included in the main. Additionally, the settings need to be changed and/or saved as well.

**Comment by shamoon** [comment]
I squashed to 1 commit and ran `pre-commit` to fix the failing checks

**Review by shamoon — APPROVED** [review]
These look fine to me

**Review by stumpylog — APPROVED** [review]

**Pull request merged**